sql-odbc/src/PowerBIConnector/SqlOdbcPBIConnector.query.pq

```diff
@@ -4,18 +4,27 @@ section SqlOdbcPBIConnector.UnitTests;
 shared MyExtension.UnitTest =
 [
     // Common variables for all tests
-    Host = "localhost",
-    Port = 9200,
+    Host = "localhost:9200",
 
     facts =
     {        
         Fact("Connection Test",  
              7,                                    
              let
-                Source = SqlOdbcPBIConnector.Contents(Host,Port),
+                Source = SqlOdbcPBIConnector.Contents(Host),
                 no_of_columns = Table.ColumnCount(Source)
              in
                 no_of_columns                
+        ),
+        Fact("calcs_data:bool0",
+            #table(type table [bool0 = logical],
+                { {null}, {false}, {true} }),
+            let
+                Source = SqlOdbcPBIConnector.Contents("localhost:9200"),
```

Comment:
Should just reference the `Host` variable.

Reply:
Fixed in a22afaca